static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js

```diff
@@ -15,32 +15,29 @@ describe('Slice Adverts', function () {
         },
         $fixtureContainer,
         injector = new Injector(),
-        sliceAdverts, config, detect, userAdPreference;
+        sliceAdverts, config, detect, commercialFeatures;
 
     beforeEach(function (done) {
         injector.test([
             'common/modules/commercial/slice-adverts',
+            'common/modules/commercial/commercial-features',
             'common/utils/config',
-            'common/utils/detect',
-            'common/modules/commercial/user-ad-preference'
+            'common/utils/detect'
         ], function () {
             sliceAdverts = arguments[0];
-            config = arguments[1];
-            detect = arguments[2];
-            userAdPreference = arguments[3];
+            commercialFeatures = arguments[1];
```

Comment:
Good idea. I've swapped the longer injections to use destructurings accordingly.